reana-ui/src/pages/launchOnReana/Welcome.module.scss

```diff
@@ -12,15 +12,47 @@
   padding: 1em 0;
 }
 
+.card-container {
+  padding-bottom: 1em;
+}
+
 .demo-card {
-  .img-wrapper > a {
-    height: 200px;
-    display: flex;
-    justify-content: center;
-  }
+  margin-bottom: 2em;
 
   :global(.ui.button).btn:hover {
     color: white;
     background-color: $sui-blue;
   }
+
+  .img-container {
+    position: relative;
+    overflow: hidden;
+  }
+
+  // Add a pseudo element to keep the aspect ratio
+  .img-container::after {
+    content: "";
+    display: block;
+    padding-bottom: calc(100% / 1); // Define ratio here
+  }
```

Comment:
I think we can avoid `.img-container::after` by using `aspect-ratio: 1` on `img-container`: https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio